signadot/operator/templates/tunnel-proxy-service.yaml

```diff
@@ -28,7 +28,5 @@ spec:
     port: 2222
   - name: revtun-xap
     port: 7777
-  - name: metrics
```

Comment:
@foxish : is this expected?

Reply:
I see a separate template for metrics. So looks like it's fine.

Reply:
Yeah, it's port has changed and it has moved to signadot/operator/templates/tunnel-proxy-metrics-service.yaml